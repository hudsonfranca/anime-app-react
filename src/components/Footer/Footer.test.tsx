import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Footer } from '.';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

let wrapper: ShallowWrapper;

beforeEach(() => {
  wrapper = shallow(<Footer />);
});

describe('<Footer/> redering', () => {
  it('Should render without errors', () => {
    const wrapper = renderer.create(<Footer />).toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
