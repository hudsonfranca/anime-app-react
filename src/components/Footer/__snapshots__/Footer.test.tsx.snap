// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> redering Should render without errors 1`] = `
.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c0 {
  grid-area: footer;
  width: 100%;
  height: 100%;
  background-color: var(--primary);
  font-size: 2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 20px;
  z-index: 0;
  position: relative;
}

.c0 small {
  color: var(--white);
  font-size: 1.6rem;
}

.c0 ul {
  width: 30%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0::before {
  content: '';
  width: 100%;
  height: 5px;
  position: absolute;
  top: 0;
  background-image: linear-gradient(to right,var(--hover1),var(--hover2));
}

.c0::after {
  content: '';
  width: 100%;
  height: 150px;
  position: absolute;
  z-index: 10;
  background-color: var(--secondary);
  -webkit-clip-path: polygon(100% 100%,74% 0,51% 100%,27% 0,0 100%);
  clip-path: polygon(100% 100%,74% 0,51% 100%,27% 0,0 100%);
  z-index: -1;
}

.c2 {
  width: 30px;
  height: 30px;
  color: var(--white);
}

.c3 {
  width: 30px;
  height: 30px;
  color: var(--white);
}

@media (max-width:736px) {
  .c0 ul {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

<div
  className="c0"
  data-test="container"
>
  <ul>
    <li>
      <img
        alt="Logo"
        src="Logo2.png"
      />
    </li>
    <li>
      <svg
        aria-hidden="true"
        className="c1 c2"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M13.397 20.997v-8.196h2.765l.411-3.209h-3.176V7.548c0-.926.258-1.56 1.587-1.56h1.684V3.127A22.336 22.336 0 0014.201 3c-2.444 0-4.122 1.492-4.122 4.231v2.355H7.332v3.209h2.753v8.202h3.312z"
        />
      </svg>
    </li>
    <li>
      <svg
        aria-hidden="true"
        className="c1 c3"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M19.633 7.997c.013.175.013.349.013.523 0 5.325-4.053 11.461-11.46 11.461-2.282 0-4.402-.661-6.186-1.809.324.037.636.05.973.05a8.07 8.07 0 005.001-1.721 4.036 4.036 0 01-3.767-2.793c.249.037.499.062.761.062.361 0 .724-.05 1.061-.137a4.027 4.027 0 01-3.23-3.953v-.05c.537.299 1.16.486 1.82.511a4.022 4.022 0 01-1.796-3.354c0-.748.199-1.434.548-2.032a11.457 11.457 0 008.306 4.215c-.062-.3-.1-.611-.1-.923a4.026 4.026 0 014.028-4.028c1.16 0 2.207.486 2.943 1.272a7.957 7.957 0 002.556-.973 4.02 4.02 0 01-1.771 2.22 8.073 8.073 0 002.319-.624 8.645 8.645 0 01-2.019 2.083z"
        />
      </svg>
    </li>
  </ul>
  <small>
    Â© Animes BR 
    2020
    . All rights reserved.
  </small>
</div>
`;
